webapp/package.json

```diff
@@ -6,91 +6,90 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "autolinker": "1.4.2",
+    "autolinker": "1.4.3",
     "bootstrap": "3.3.7",
     "bootstrap-colorpicker": "2.3.6",
     "chart.js": "2.5.0",
     "compass-mixins": "0.12.10",
     "fastclick": "1.0.6",
     "flux": "3.1.2",
     "font-awesome": "4.7.0",
-    "highlight.js": "9.10.0",
+    "highlight.js": "9.11.0",
     "inobounce": "0.1.4",
     "intl": "1.2.5",
     "jasny-bootstrap": "3.1.3",
-    "jquery": "3.1.1",
+    "jquery": "^3.2.1",
     "localforage": "1.5.0",
     "marked": "mattermost/marked#8f5902fff9bad793cd6c66e0c44002c9e79e1317",
     "match-at": "0.1.0",
     "mattermost-redux": "mattermost/mattermost-redux#webapp-master",
     "object-assign": "4.1.1",
-    "pdfjs-dist": "1.7.363",
-    "perfect-scrollbar": "0.6.16",
-    "prop-types": "15.5.9",
-    "react": "15.4.2",
-    "react-addons-pure-render-mixin": "15.4.2",
-    "react-bootstrap": "0.30.8",
-    "react-custom-scrollbars": "4.0.2",
-    "react-dom": "15.4.2",
-    "react-intl": "2.2.3",
+    "pdfjs-dist": "^1.8.361",
```

Comment:
It doesn't matter because yarn uses the yarn.lock file. But why not make it look nice with that switch anyway.